source/includes/nuget-versioned-install/1.0.0.rst

```diff
@@ -0,0 +1,3 @@
+.. code-block:: shell
+
+   Install-Package MongoDB.Analyzer -Version 1.0.0
```

Comment:
question: Why define constants if you aren't using them here? It looks like you may have meant to?

Reply:
I was under the impression that constants were not replaced in code blocks but this seems to not be the case. Good catch!

Removed includes and replaced with code snippet that uses constants.